packages/nocodb/src/lib/services/syncService/helpers/job.ts

```diff
@@ -1406,7 +1413,7 @@ export default async (
         case UITypes.DateTime:
         case UITypes.CreateTime:
         case UITypes.LastModifiedTime:
-          rec[key] = dayjs(value).format('YYYY-MM-DD HH:mm');
+          rec[key] = dayjs(value).utc().format('YYYY-MM-DD HH:mm');
```

Comment:
as i rmb adding `utc()` would produce wrong result. The date `2023-01-01` would become `2022-12-31`.